utils/navigator_msgs/msg/PerceptionObject.msg

```diff
@@ -8,16 +8,21 @@ string IDENTIFY_AND_DOCK=dock
 string SCAN_THE_CODE=scan_the_code
 string TOTEM=totem
 string START_GATE_BUOY=start_gate
+string BUOY=buoy
 string UNKNOWN=unknown
+string ALL=all
 
 geometry_msgs/Point position      # Estimated position of the buoy stamped (in the frame of the header)
 
 uint32 id                         # If we want to label each buoy and keep track of it
 uint8 confidence                  # 0-255 how confident are we that this is a buoy
 
-float64 height                    # Height and width for the detected object
-float64 width                     # Can be used for detecting different objects
-float64 depth					  # Third dimension in x,y,z...
+float64 zHeight                    # Height and width for the detected object
+float64 xWidth                     # Can be used for detecting different objects
+float64 yDepth					  # Third dimension in x,y,z...
```

Comment:
Has @rleyva talked to you about including a PointCloud2 in this? That could replace intensity and points.

Also, what do you think about combining zHeight, xWidth, and yDepth into one object such as a geometry_msgs/Vector3 to represent the size to keep the message clean?